manimlib/mobject/svg/text_mobject.py

```diff
@@ -314,86 +307,180 @@ def text2svg(self):
             if setting.line_num != last_line_num:
                 offset_x = 0
                 last_line_num = setting.line_num
-            context.move_to(START_X + offset_x, START_Y + lsh*setting.line_num)
+            context.move_to(START_X + offset_x, START_Y + line_spacing * setting.line_num)
             context.show_text(text)
             offset_x += context.text_extents(text)[4]
+        surface.finish()
+        return file_name
+
+
+class TextWithBackground(Text):
```

Comment:
docs?

Reply:
I will. Firstly i supposed to learn from your suggestions on documentions of Code().

Reply:
got it!